packages/react-email/source/index.ts

```diff
@@ -13,4 +14,11 @@ program
   .description('Starts the application in development mode')
   .action(dev);
 
+program
+  .command('build')
```

Comment:
I wonder if we should call this command `export` instead of `build`. I feel like with `export` there's a clear expectation that some file will be generated from that. What do you think?

Reply:
Yep that makes more sense. Will do.

Reply:
Agree! Let's change that to `export`.